### PR TITLE
Developing `wandb resume run` function

### PR DESCRIPTION
**Pull request by manhdung20112000**
A `test_cli` quick pipeline sketch

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -2,6 +2,33 @@
...
+    path = 'path/to/dataset'
+    create_dataset_artifact(path, dataset_name='demo-dataset')
+
+    # delete local dataset
+    down_path = wb_logger.download_dataset_artifact(dataset_name='demo-dataset')
+
+    # load dataset
+
+    # setup a simple training run
+    model = Net()
```
Comment (gungui98) [inline tests/test_cli.py:21]: không cần train, mình k test chức năng train

File: tests/test_cli.py [hunk 2]
```diff
@@ -2,6 +2,33 @@
...
+    for epoch in range(100):
+        torch.save(model.state_dict(), 'model.pth')
+        wb_logger.log_model('model.pth')
+    
+    # delete current model weight
+
+    # attempt to download latest weight and continues train
+    weight_path = download_model_artifact('experiment/model.pth')
+
```
Comment (gungui98) [inline tests/test_cli.py:30]: thêm check file có đúng không bằng os.exists

**Comment by manhdung20112000** [comment]
# New update
## Wandb organization in `SummaryWriter`
`SummaryWriter` now can be selected and logged with the `Wandb` team account. 
I've created a `Wandb` team named [`uet-ailab`](https://wandb.ai/uet-ailab) for everyone can join in.
Example:
- Old
```python
logger = uetai.SummaryWriter(log_dir='uetai-demo')
```
- New
```python
# optional
logger = uetai.SummaryWriter(log_dir='uetai-demo', entity='uet-ailab')
```
## Select `wandb` or `tensorboard` 
When initializing `SummaryWriter` we can specify which logger we want to use (or let the library decides if no specific logger need) by args `log_tool` (`wandb` or `tensorboard`)
```python
logger = uetai.SummaryWriter(log_dir='uetai-demo', log_tool='wandb')
```
Notes: If `wandb` was choice (while initializing) but no package named `wandb` was found, it will be installed automatically with `pip`

## Download by URL and automate `unzip` it
We provide `data_path` which can be used to select specific `wandb` dataset artifact to be downloaded. Now it can be used to download through URL
```python
dataset_dir = logger.data_path(path="https://github.com/manhdung20112000/torch-simple-logger/releases/download/dataset_v1.0/MNIST.zip")
```
Notes: Automate unzip (if the URL link to `zip` file) and save in the dir with `zip` file name

**Comment by manhdung20112000** [comment]
This branch has already been merged though #9

**Pull request closed without merging**